sources/packages/backend/apps/test-db-seeding/src/db-seeding/aest/basic-seeding/create-aest-users.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { User } from "@sims/sims-db";
+import { createFakeUser } from "@sims/test-utils";
+import {
+  DataSeed,
+  DataSeedMethod,
+  SeedPriorityOrder,
+} from "../../../seed-executors";
+import { Repository } from "typeorm";
+import * as faker from "faker";
+
+@Injectable()
+@DataSeed({ order: SeedPriorityOrder.Priority1 })
+export class CreateAESTUsers {
+  constructor(
+    @InjectRepository(User)
+    private readonly userRepo: Repository<User>,
+  ) {}
+
+  /**
+   * Create AEST users used for tests.
+   */
+  @DataSeedMethod()
+  private async createAESTUsers(): Promise<void> {
+    await this.createAESTUser(
+      process.env.E2E_TEST_AEST_BUSINESS_ADMINISTRATORS_USER,
+    );
+    await this.createAESTUser(process.env.E2E_TEST_AEST_OPERATIONS_USER);
+    await this.createAESTUser(
+      process.env.E2E_TEST_AEST_OPERATIONS_ADMINISTRATORS_USER,
+    );
+    await this.createAESTUser(process.env.E2E_TEST_AEST_MOF_OPERATIONS_USER);
+    await this.createAESTUser(process.env.E2E_TEST_AEST_READ_ONLY_USER);
+  }
+
+  /**
+   * Create a fake AEST user persisted in the DB.
+   * @param userName username of the AEST user to be created.
+   */
+  private async createAESTUser(userName: string): Promise<void> {
+    // Create fake user.
+    const fakeUser = createFakeUser(userName);
+    fakeUser.firstName = faker.name.firstName();
```

Comment:
Is there a reason to `faker.name.firstName()` be used on lines 44 and 45?
Besides that, the `createFakeUser` is already setting these properties, is there a reason to set them again?

Reply:
Yes, you're right. Sorry. Fixing it.